eng/testing/RunnerTemplate.cmd

```diff
@@ -92,6 +92,45 @@ if %_exit_code%==1 (
     )
   )
 )
+
+if NOT DEFINED __IsXUnitLogCheckerSupported (
+  echo The '__IsXUnitLogCheckerSupported' env var is not set.
+  GOTO SKIP_XUNITLOGCHECKER
+)
+if NOT %__IsXUnitLogCheckerSupported%==1 (
+  echo XUnitLogChecker not supported for this test case. Skipping.
+  GOTO SKIP_XUNITLOGCHECKER
+)
```

Comment:
Why do you need these different cases? Isn't one condition enough?

Reply:
I can remove the first case. I just noticed the "set" command for this env var is verbose in the logged file, so if it does not get set, we won't see it.